docs/core/deploying/index.md

```diff
@@ -46,22 +46,25 @@ Cross-platform binaries can be run on any operating system as long as the target
 
 The following command produces a cross-platform binary:
 
-| Type                                                                                 | SDK 2.1 | SDK 3.x | SDK 5.0 | SDK 6.0 | Command |
-| -----------------------------------------------------------------------------------  | ------- | ------- | ------- | ------- | ------- |
-| [framework-dependent cross-platform binary](#publish-framework-dependent).           | ✔️      | ✔️      | ✔️      | ✔️      | [`dotnet publish`](../tools/dotnet-publish.md) |
+| Type                                                                                | Command |
+| ----------------------------------------------------------------------------------- | ------- |
+| [framework-dependent cross-platform binary](#publish-framework-dependent).          | [`dotnet publish`](../tools/dotnet-publish.md) |
 
 ## Publish framework-dependent
```

Comment:
```suggestion
## Framework-dependent
```

Reply:
To be consistent with the heading on line 61.

Reply:
The section below (unedited in the PR) is `Publish self-contained`. The section on line 61 is a subsection of this one, describing the behavior of "platform-specific" feature of framework-dependent.